pkg/cmd/setup.go

```diff
@@ -63,6 +63,12 @@ func setup(cmd *cobra.Command, args []string) {
 		utils.LogDebugError(err.Unwrap())
 	}
 
+	// prompt whether to use repoConfig
+	useRepoConfig := true
+	if promptUser && (repoConfig.Setup.Project != "" || repoConfig.Setup.Config != "") {
+		useRepoConfig = utils.ConfirmationPrompt("Read values from repo config file (doppler.yaml)?", true)
```

Comment:
`Use default setting from repo config file (doppler.yaml)?`?

Reply:
I like that